Stripe/PublicHeaders/STPBankAccount.h

```diff
@@ -10,85 +10,108 @@
 
 #import "STPBankAccountParams.h"
 #import "STPAPIResponseDecodable.h"
+#import "STPSourceProtocol.h"
 
+NS_ASSUME_NONNULL_BEGIN
 
 /**
  Possible validation states for a bank account.
-
- - STPBankAccountStatusNew: The account has had no activity or validation performed
- - STPBankAccountStatusValidated: Stripe has determined this bank account exists.
- - STPBankAccountStatusVerified: Bank account verification has succeeded.
- - STPBankAccountStatusVerificationFailed: Verification for this bank account has failed.
- - STPBankAccountStatusErrored: A transfer sent to this bank account has failed.
  */
 typedef NS_ENUM(NSInteger, STPBankAccountStatus) {
+    /**
+     The account has had no activity or validation performed
+     */
     STPBankAccountStatusNew,
+
+    /**
+     Stripe has determined this bank account exists.
+     */
     STPBankAccountStatusValidated,
+
+    /**
+     Bank account verification has succeeded.
+     */
     STPBankAccountStatusVerified,
+
+    /**
+     Verification for this bank account has failed.
+     */
     STPBankAccountStatusVerificationFailed,
+
+    /**
+     A transfer sent to this bank account has failed.
+     */
```

Comment:
Yeah after playing around with it a bit, SourceKit (the Apple framework both Xcode and jazzy use for documentation) only recognizes enum docs written out this way, even though the Xcode doc generator doesn't do this. I'm gonna change everything to this as I audit the rest of our docs and will update our style guide.